### PR TITLE
Add close button to open form

### PR DESCRIPTION
**Pull request by kseniaSs**
Add a close button on the top left to open form

Снимок экрана 2023-01-26 в 10 07 52
 
Please check the design. I changed the top padding

### CODE REVIEW

File: src/utils/createField.ts [hunk 1]
```diff
@@ -43,6 +43,31 @@ export function createSubmit(): HTMLElement {
...
+ * @returns {HTMLButtonElement}
+ */
+export function createClose(): HTMLButtonElement {
+  const closeContainer = make('button', classes.close) as HTMLButtonElement;
+
+  addBackgroundIcon(closeContainer, IconCross);
+
+  return closeContainer;
+}
+
```
Comment (TatianaFomina) [inline src/utils/createField.ts:70]: close button is not a field, seems like this file is not the best place to define `createClose` function.

File: src/utils/createField.ts [hunk 2]
```diff
@@ -43,6 +43,31 @@ export function createSubmit(): HTMLElement {
...
+/**
+ * Helper for creating close button
+ * 
+ * @returns {HTMLButtonElement}
+ */
+export function createClose(): HTMLButtonElement {
+  const closeContainer = make('button', classes.close) as HTMLButtonElement;
+
+  addBackgroundIcon(closeContainer, IconCross);
```
Comment (TatianaFomina) [inline src/utils/createField.ts:66]: Why not just insert icon as `innerHTML`?

```
closeContainer.innerHTML = IconCross;
```

File: src/utils/createField.ts (hunk 1 above)
Comment (TatianaFomina) [reply src/utils/createField.ts:70]: I would suggest moving this logic into Form class, and also put attaching click event listener to the same method

**Comment by neSpecc** [comment]
Added cross button to the [Figma](https://www.figma.com/file/iuW5mRYvABhr36YpKqPjxP/CodeX-Survey?node-id=1%3A12&t=7NOgZdicwTwy2qfs-4):

image

**Review by TatianaFomina — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**